tcms/testcases/views.py

```diff
@@ -683,15 +707,16 @@ class TestCaseSimpleCaseRunView(TemplateView):
     """
 
     template_name = 'case/get_details_case_case_run.html'
+    caserun_id = None
 
     # what permission here?
-    def get(self, request, case_id):
+    def get(self, request, *args, **kwargs):
```

Comment:
I think this is breaking the tests. Maybe revert it so everything passes and we can deal with this issue later.